libraries/helpers.rb

```diff
@@ -0,0 +1,29 @@
+module OSLMatrix
+  module Cookbook
+    module Helpers
+
+      # Create the synapse user, and the recommended directories
+      def init_environment()
```

Comment:
Please prepend method names with `osl_matrix_` so we don't run into conflicts elsewhere.

Reply:
The helpers function was for a previous implementation, and is no longer required. I've removed the file.